Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17 AS builder
 WORKDIR /go/src/github.com/openshift/shared-resources-operator
 COPY . .
-COPY vendor/github.com/openshift/api/sharedresource/v1alpha1/*.crd.yaml assets/
+COPY vendor/github.com/openshift/api/sharedresource/v1alpha1/zz_generated.crd-manifests/*.crd.yaml assets/
```

Comment:
The files have different names now. I would update COPY command here to rename them to the expected names. The operator logs
```
F0805 15:13:16.584002       1 cmd.go:179] error occurred reading file "0000_10_sharedsecret.crd.yaml": open 0000_10_sharedsecret.crd.yaml: file does not exist
```